Kitodo/src/test/java/org/kitodo/services/data/BatchServiceIT.java

```diff
@@ -36,8 +37,9 @@ public static void prepareDatabase() throws DAOException, IOException, ResponseE
     }
 
     @AfterClass
-    public static void cleanDatabase() {
+    public static void cleanDatabaseAndIndex() throws IOException, ResponseException{
```

Comment:
How can a no-op method throw an Exception and why is this code comment out? If this method is not really used or not providing any functionality then it should be removed. Method place holders are confusing.